### PR TITLE
Feature/module 4 components

### PR DESCRIPTION
**Pull request by UltramarineDev**

### CODE REVIEW

**Comment by UltramarineDev** [comment]
Planned to make one more push with styles ~ in 14:00.

File: netflix-roulette/src/components/features/movie-form/MovieForm.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './MovieForm.module.scss';
+
+const MovieFormComponent = () => (
+  <>
+    <form className={styles.form}>
+      <p className={styles.label}>TITLE</p>
+      <input></input>
```
Comment (omelenetskiy) [inline netflix-roulette/src/components/features/movie-form/MovieForm.js:9]: `<input type="text" placeholed="...." />`

File: netflix-roulette/src/components/features/movie-form/MovieForm.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <p className={styles.label}>RELEASE DATE</p>
+      <input></input>
+      <p className={styles.label}>MOVIE URL</p>
+      <input></input>
+      <p className={styles.label}>GENRE</p>
```
Comment (omelenetskiy) [inline netflix-roulette/src/components/features/movie-form/MovieForm.js:14]: constants -> relatedName.js

```
export const categories = {
RELEASE DATE: 'RELEASE DATE'
}
```

File: netflix-roulette/src/components/shared/modal/Modal.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '/src/components/shared/button';
+import { BUTTON_TYPES } from '/src/components/shared/constants';
+
+import styles from './Modal.module.scss';
+
+const ModalComponent = ({ onClose, isOpen, children, confirmLabel, onConfirm, className }) => (
+  <>
```
Comment (omelenetskiy) [inline netflix-roulette/src/components/shared/modal/Modal.js:10]: https://reactjs.org/docs/portals.html

File: netflix-roulette/src/components/shared/modal/Modal.module.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+.modal {
+  position: fixed;
+  z-index: 2;
+  left: 0;
+  top: 0;
+  width: 100%; 
+  height: 100%; 
+  overflow: auto;
+  background-color: rgb(0,0,0);
```
Comment (omelenetskiy) [inline netflix-roulette/src/components/shared/modal/Modal.module.scss:9]: useless

File: netflix-roulette/src/components/shared/movie-card/MovieCard.js [hunk 5]
```diff
@@ -1,11 +1,35 @@
...
+  genre, 
+  year, 
+  onActionsClick, 
+  showOptions, 
+  actions, 
+  onActionClick,
+ }) => (
   <>
     <div className={styles.cardWrapper}>
+        <a onClick={onActionsClick} className={styles.dotsMenu}>
```
Comment (omelenetskiy) [inline netflix-roulette/src/components/shared/movie-card/MovieCard.js:20]: Let's move `onActionsClick` to local state

**Review by omelenetskiy — APPROVED** [review]

**Pull request merged**